FirebaseMessaging/Apps/README.md

```diff
@@ -0,0 +1,4 @@
+### Firebase Messaging Test Apps
+
+We offer two test apps: one in the Sample folder is a simple test app with the minimum Firebase Messaging code you need to enable delivering notifications. We kept the code minimum for you to easy debug and compare the sample code. The one inside the AdvancedSample folder is a test app based on the test app in the Sample folder. Actually they share most of the code in the main target. The AdvancedSample app provides advanced app extensions features, such as notification service extensions, watchkit, and app clips. So this app has more targets than the Sample app and requires your own provisioning profiles and certificates to be setup so that it can be run on a real device.
```

Comment:
Keep line limit to 100 so the README is readable in any editor

Reply:
s/minimum for you to easy/minimal for you to easily/

Reply:
Done.